src/components/internal/PersonalDetails/validate.ts

```diff
@@ -1,17 +1,20 @@
-import { email, telephoneNumber } from '../../../utils/regex';
-import { unformatDate } from '../FormFields/InputDate/utils';
+import {email, telephoneNumber} from '../../../utils/regex';
+import {unformatDate} from '../FormFields/InputDate/utils';
+import ValidationRule from "../../../utils/Validator/ValidationRule";
+import Language from "../../../language/Language";
 
+const i18n = new Language();
```

Comment:
In practice - we would also need to access the existing Language instance, rather than creating a new one.
Creating a new one like this will default to `en-US` and so only give keys in that language. 
In the future we may want the merchant to pass in the translations object to use - then we would _definitely_ have to make sure to access the existing Language instance.

Reply:
Fixed. Thanks for that. Did not know.